lib/ecto_psql_extras.ex

```diff
@@ -67,13 +70,27 @@ defmodule EctoPSQLExtras do
     end
   end
 
-  defp pg_stat_statements_version(repo) do
+  def pg_stat_statements_version(repo) do
     case query!(repo, @pg_stat_statements_query) do
       %{rows: [[value]]} when is_binary(value) -> Postgrex.Utils.parse_version(value)
       _ -> nil
     end
   end
 
+  defp ssl_used_query(repo) do
+    case repo && ssl_info_enabled(repo) do
+      true -> %{ssl_used: EctoPSQLExtras.SSLUsed}
+      _ -> %{}
+    end
```

Comment:
Just nitpick of style: here we could use an `if` statement for easier readability. WDYT?

```suggestion
    if repo && ssl_info_enabled(repo) do
      %{ssl_used: EctoPSQLExtras.SSLUsed}
    else
      %{}
    end
```